src/components/App.tsx

```diff
@@ -1374,6 +1376,19 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         isCollaborating: true,
         isLoading: opts.showLoadingState ? true : this.state.isLoading,
       });
+
+      try {
+        const elements = await loadFromFirebase(roomId, roomSecret);
+        if (elements) {
+          updateScene(
+            { type: "SCENE_INIT", payload: { elements } },
```

Comment:
There seems to be an issue where this takes priority over the peer updates.

Client A: connect
Client B: connect
Client A: draw a rectangle, draw a circle on the right of the rectangle
Wait for the push to firebase
Client A: move the circle above the rectangle
Client B: reload the page
Client B sees the circle on the right instead of the top
Wait for the push to firebase
Client B sees the circle on the top.

Reply:
removed `init: true`